app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -444,6 +447,19 @@ def mail_delivery_params(proofed_at)
     return { wait_until: wait_until, queue: :intentionally_delayed }
   end
 
+  def send_enrollment_status_sms_notification(enrollment:)
+    return unless IdentityConfig.store.in_person_send_proofing_notifications_enabled
+    return if enrollment&.proofed_at.blank?
+    sms_delay_hours = IdentityConfig.store.in_person_results_delay_in_hours ||
+                      DEFAULT_EMAIL_DELAY_IN_HOURS
+    wait_until = enrollment.proofed_at + sms_delay_hours
+    if wait_until <= Time.zone.now
```

Comment:
Shouldn't we be checking to see if wait_until is in the future?

```suggestion
    if wait_until > Time.zone.now
```

Reply:
Good catch, fixed.